Node Status Monitor/Python v2.7/api_helper.py

```diff
@@ -0,0 +1,111 @@
+import requests
+import time
+import urllib2
+import urllib
+import json
+import logging
+from configparser import ConfigParser
+import os.path
+from os import path
+
+# congiguration
+logging.basicConfig(filename='app.log',level=logging.INFO,format='%(asctime)s.%(msecs)03d %(levelname)s %(module)s - %(funcName)s: %(message)s',
+    datefmt='%Y-%m-%d %H:%M:%S')
+config = ConfigParser()
+config.read('config.cfg')
+# ---------Files----------
+new_data_file=config.get('files','new_data_file')
+old_data_file=config.get('files','old_data_file')
+result_file=config.get('files','result_file')
+
+# Api to get token
+def gen_token(token_url,client_key, client_secret):
```

Comment:
Should name consistently with the javascript files for similar functionality.

Reply:
Code Updated